.editorconfig

```diff
@@ -149,3 +149,9 @@ dotnet_diagnostic.CA2254.severity = suggestion
 
 # Extract strings to resource table (the solution will not be localized).
 dotnet_diagnostic.CA1303.severity = none
+
+# Use compound assignment
```

Comment:
Please revert these changes before merging. VS 2022 (non preview) is not complaining. Otherwise, please create an additional PR to fix rather than suppressing